### PR TITLE
More about load balancing, better formatting.

### PR DESCRIPTION
**Pull request by svenakela**
The contribution about load balancing was messing up the readme, I moved some text and added more info.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,26 +162,35 @@ changing this value will affect all.
...
-blueprint you'll get automatic load balancing for for your charger (i.e. the 
-charger limit is updated constantly to avoid fuse overload).
+blueprint you'll get automatic load balancing for your charger (i.e. the charger 
+limit is updated constantly to avoid fuse overload).
+
+The automation created with the blueprint manages current limiting only. It will check 
+the charger status and sets the current accordingly. If charging is enabled and 
+possible it will manage the limit. It will _not_ maintain the charging nor scheduling.
+
+How setup the automation, how logic works and what all settings mean is documented in the [blueprint readme](https://github.com/svenakela/ha/blob/main/zaptec/README.md).
```
Comment (philsson) [inline README.md]: How ***to*** setup...

**Comment by sveinse** [comment]
I have one concern: I think its important that the features or implementations of the load balancing blueprint is not tied too tightly with this project. If we do, this will result in this repo having to update whenever new features were changed in the blueprint repo. So I think the readme should give very general information that the load balancing blueprint exists and then refer to the blueprint repos readme.

**Comment by svenakela** [comment]
> I have one concern: I think its important that the features or implementations of the load balancing blueprint is not tied too tightly with this project. If we do, this will result in this repo having to update whenever new features were changed in the blueprint repo. So I think the readme should give very general information that the load balancing blueprint exists and then refer to the blueprint repos readme.

I don't mind, feel free to remove as much as you like. Maybe even move to the bottom of the readme as "Usage Examples" or similar?

**Comment by svenakela** [comment]
@sveinse had some spare time to adjust the readme. I've shortened the text and moved it to the bottom of the file.

**Comment by svenakela** [comment]
@sveinse is this PR stale? The updates are at least much better than the previously approved adds.

**Pull request merged**